docs/08-developer-guide/02-run/02-docker-compose.md

```diff
@@ -10,3 +10,143 @@ sidebar_label: Docker Compose
 - Define the Ozone runner image at `ozone-docker-runner` and its purpose to wrap Ozone binaries.
 - How to define which image gets used in the cluster (runner or other pre-built image).
 - Changing configurations in Docker Compose (`docker-config` file, transformation.py, etc).
+
```

Comment:
We can delete the TODO above this section since we are filling in the page in this PR.

Reply:
Fixed